samples/dnn/models.yml

```diff
@@ -178,3 +178,16 @@ fcn8s:
   height: 500
   rgb: false
   sample: "segmentation"
+
+fcnresnet101:
+  load_info:
+    url: "https://github.com/richard28039/fcnresnet101_onnx_model/raw/main/fcn_resnet101.onnx?download="
+    sha1: "1242104e22c9eb0b320d7ae36b11cdd1fda40857"
+  model: "fcn_resnet101.onnx"
+  mean: [103.5, 116.2, 123.6]
+  scale: 0.019
+  width: 500
+  height: 500
+  rgb: false
+  classes: "pascal-voc-classes.txt"
```

Comment:
`pascal-voc-classes.txt` missed file

Reply:
Hello sir, classes file is here https://github.com/richard28039/fcnresnet101_onnx_model/blob/main/pascal-voc-classes.txt

Reply:
Please use https://github.com/opencv/opencv/blob/4.x/samples/data/dnn/object_detection_classes_pascal_voc.txt instead. They match each other.